src/families/tezos/datasets/tezos.scanAccounts.1.js

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
this file needs to be imported from the main test-dataset (see in similar family) otherwise it's not going to do anything.